### PR TITLE
DEVDOCS-4631: [bug fix] Specify server URLs for Storefront APIs

### PR DESCRIPTION
**Pull request by bc-traciporter**
[DEVDOCS-4631](https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4631)
See slack conversation: https://bigcommerce.slack.com/archives/C57SQ401K/p1668115831240039

## What changed?
changed server URLs for storefront APIs

## Anything else?
Related PRs, salient notes, etc

### CODE REVIEW

**Comment by bc-andreadao** [comment]
- I think the [Storefront Orders](https://developer.bigcommerce.com/api-reference/f40f1f75f3e00-get-order) should be changed also (people can't all be making calls to the same https://api.bigcommerce.com without telling the API which store). @bc-tgomez not sure if you had thoughts on how it got in there? Or if we should check with your SMEs?
- One additional thing, I wondering what are people's thoughts about adding a note for each endpoint that says "The `yourstore.example.com` is the store domain specific to your storefront" ? I got the idea from my Shipping Provider API (screenshot attached). @slsriehl if you have thoughts about wording

screenshot

File: reference/carts.sf.yml [hunk 1]
```diff
@@ -19,8 +22,10 @@ paths:
         Returns a *Cart*.
 
         <!-- theme: info -->
+
         > #### Note
-        > The Send a Test Request feature is not currently supported for this endpoint.
+        > * The `yourstore.example.com` is the store domain specific to your storefront.
```
Comment (slsriehl) [inline reference/carts.sf.yml:27]:
```suggestion
        > * The URL `yourstore.example.com` is the store domain specific to your storefront. 
```

File: reference/subscriptions.sf.yml [hunk 2]
```diff
@@ -57,8 +61,7 @@ components:
           description: Has subscriber provided consent for receiving Marketing emails.
         acceptsAbandonedCartEmails:
           type: boolean
-          description: Has subscriber provided consent for receiving Abandon Cart
-            emails.
+          description: Has subscriber provided consent for receiving Abandon Cart emails.
```
Comment (slsriehl) [inline reference/subscriptions.sf.yml:64]:
```suggestion
          description: Describes whether subscriber has consented to receive Abandoned Cart emails.
```

File: reference/subscriptions.sf.yml [hunk 3]
```diff
@@ -57,8 +61,7 @@ components:
           description: Has subscriber provided consent for receiving Marketing emails.
```
Comment (slsriehl) [inline reference/subscriptions.sf.yml:61]:
```suggestion
          description: Describes whether subscriber has consented to receive Marketing emails.
```

File: reference/carts.sf.yml (hunk 1 above)
Comment (bc-traciporter) [reply reference/carts.sf.yml:27]: @bc-andreadao Ok, sounds good. I am working on this now. I will also update Shipping Provider API.

File: reference/shipping_provider.yml [hunk 4]
```diff
@@ -53,8 +53,7 @@ paths:
 
         > #### Note
 
-        > * The `your_app.example.com` and `rate` are the host and path specific to the shipping provider. 
-
+        > * Substitute the host and path specific to the shipping provider for  `your_app.example.com` and `rate`.
```
Comment (bc-andreadao) [inline reference/shipping_provider.yml:56]:
```suggestion
        > * Substitute the host and path specific to the shipping provider for `your_app.example.com` and `rate`. 
```

**Review by bc-andreadao — APPROVED** [review]

**Review by markcmurphy — APPROVED** [review]

**Pull request merged**